docs/en/02_Developer_Guides/05_Extending/05_Injector.md

```diff
@@ -111,6 +111,16 @@ Now the dependencies will be replaced with our configuration.
 	echo ($object->textProperty == 'My Text Value');
 	// returns true;
 
+As well as properties, method calls can also be specified:
+
+	:::yml
+	Injector:
+	  Logger:
+	    class: Monolog\Logger
+	    methods:
+	      - [ pushHandler: [ %$DefaultHandler ] ]
```

Comment:
oh, yes, I see now. i'll fix.

note that i've opted to copy Symfony's syntax for these.